ntbs-service-unit-tests/Helpers/NameFormattingHelperTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using ntbs_service.Helpers;
+using Xunit;
+
+namespace ntbs_service_unit_tests.Helpers
+{
+    public class NameFormattingHelperTest
+    {
+        [Fact]
+        public void FormatDisplayName_FormatsSingleName()
```

Comment:
I think you should commonise these tests as they are all the same. There are two options:
1. Keep all the individual tests and replace the body with a single call to a common method (e.g. AssertDisplayNameMatchesExpected or something like that)
2. Have a single test marked with the `Theory` and `InlineData` attributes, to pass in the input and expected values.

In a case like this I would probably prefer the latter for the sake of clarity, but you do lose the names of the tests which are providing some value documenting the behaviour. So it's up to you!

Reply:
Yeah I agree. I've gone with the second approach in 0f7b168